Can't compile in Angular 7 project - error TS7006: Parameter 'file' implicitly has an 'any' type.

I am getting the error `ERROR in node_modules/ngx-file-helpers/src/file-picker.directive.d.ts(10,14): error TS7006: Parameter 'file' implicitly has an 'any' type.` when compiling my project after including ngx-file-helpers.

The solution was to just modify the file in question:
```
import { ElementRef, EventEmitter, OnInit, Renderer2 } from '@angular/core';
import { ReadFile } from './read-file';
import { ReadMode } from './read-mode.enum';
export declare class FilePickerDirective implements OnInit {
    private el;
    private renderer;
    accept: string;
    multiple: any;
    readMode: ReadMode;
    filter: (file: any) => boolean; // <--- change this line here
    filePick: EventEmitter<ReadFile>;
    readStart: EventEmitter<number>;
    readEnd: EventEmitter<number>;
    private _multiple;
    private input;
    constructor(el: ElementRef, renderer: Renderer2);
    ngOnInit(): void;
    reset(): void;
    browse(): void;
    private readFile(file);
}
```

I tried setting my tsconfig strict mode off, but that didn't solve it.

--- a/src/file-picker.directive.ts
+++ b/src/file-picker.directive.ts
@@ -30,7 +30,7 @@ export class FilePickerDirective implements OnInit {
 
   @Input('ngxFilePicker') readMode: ReadMode;
 
-  @Input() filter: (file) => boolean = () => true;
+  @Input() filter: (file: any) => boolean = () => true;
 
   @Output() public filePick = new EventEmitter<ReadFile>();
 

